kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -149,13 +150,31 @@ def _replace(term: KInner) -> KInner:
         return CTerm(config=bottom_up(_replace, cterm.config), constraints=cterm.constraints)
 
     def custom_step(self, cterm: CTerm) -> KCFGExtendResult | None:
+        """Given a CTerm, update the JUMPDESTS_CELL and PROGRAM_CELL if the rule 'EVM.program.load' is at the top of the K_CELL.
```

Comment:
Based on your measurements, this does not seem to improve performance. What's the purpose of this change?

Reply:
At this moment we cannot compute jump destinations for bytecodes that have symbolic values, blocking issues such as https://github.com/runtimeverification/kontrol/pull/573 and https://github.com/runtimeverification/evm-semantics/issues/2219.

Reply:
I see, thank you. Please include these details in the PR description.